site/app/libraries/database/DatabaseQueries.php

```diff
@@ -9414,4 +9461,27 @@ public function removeDockerImageOwner(string $image, User $user): bool {
         }
         return $this->submitty_db->getRowCount() > 0;
     }
+
+    /**
+     * @param Gradeable $gradeable
+     * @param array<Redaction> $redactions
+     */
+    public function updateRedactions(Gradeable $gradeable, array $redactions): void {
```

Comment:
What do you think about using Doctrine for this new relation instead?

Reply:
This is the only place that they are currently queried, is there a way to use doctrine for just part of this query? https://github.com/Submitty/Submitty/blob/ea63ed2f0347bc45522018e51969466fece739d3/site/app/libraries/database/DatabaseQueries.php#L5888-L5899